pkgs/by-name/fr/framework-led-matrix-monitor/package.nix

```
@@ -0,0 +1,81 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "framework-led-matrix-monitor";
+  version = "1.0.0";
```

```suggestion
  version = "1.0.0-unstable-2025-07-28";
```
and the version will need to change to this